torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -110,7 +110,10 @@ int64_t SizeEq::getDynamicValue() const {
 
 std::string SizeEq::ToString() const { return "SizeEq"; }
 
-SizeConstant::SizeConstant(int64_t val) : Scalar(c10::Scalar{val}, xla::S64){};
+SizeConstant::SizeConstant(int64_t val)
+    : Scalar(c10::Scalar{val},
+             xla::ShapeUtil::MakeShape(
+                 GetShapeDimensionType(/*device=*/nullptr), {})){};
```

Comment:
can you update this to all size nodes?

Reply:
I'm planning to add other missing python test for other node in later PR and at that time I'd have to update it. How about I update it in the later PR?